onto_tool/ontograph.py

```diff
@@ -741,7 +743,11 @@ def create_instance_graf(self, data_dict=None):
             return
 
         # Determine the maximum number any edge occurs in the data, so the edge widths can be properly scaled
-        max_common = max(occurs for class_data in data_dict.values() for occurs in class_data['links'].values())
+        common_list = [
+            occurs for class_data in data_dict.values()
+            for occurs in class_data['links'].values()
+        ]
+        max_common = 0 if len(common_list) == 0 else max(common_list)
```

Comment:
Sometimes the int list created inside `max` will be of length 0. Since `max` requires a list with at least 1 item, the script fails.

Reply:
Changing the value set if `len(common_list) == 0` from `0` to any other value seems to have no affect on the output image. It might make more sense to make it `1` with respect to the above comment, but not sure.